code/game/machinery/medical_kiosk.dm

```diff
@@ -213,16 +213,17 @@
 	var/brain_status = "Brain patterns normal."
 	if(LAZYLEN(altPatient.get_traumas()))
 		var/list/trauma_text = list()
-		for(var/datum/brain_trauma/B in altPatient.get_traumas())
+		for(var/t in altPatient.get_traumas())
+			var/datum/brain_trauma/traumas = t
```

Comment:
Good on the var name, B is really dumb, this is real minor but traumas should be trauma